synapse/local_infer.go

```diff
@@ -264,3 +264,14 @@ func (s *Synapse) download(infohash string, request uint64) error {
 
 	return nil
 }
+
+func (s *Synapse) seedingLocal(filePath string, isLinkMode bool) (ih string, err error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+	ih, err = s.config.Storagefs.SeedingLocal(ctx, filePath, isLinkMode)
+	if err != nil {
```

Comment:
The interface should throw KERNEL RUNTIME ERROR, but it's considerable to
log the necessary error information for debugging, try to print the error
instead of swallowing it.